README.md

```diff
@@ -20,6 +20,11 @@ contributions.
 Installation
 ------------
 
+### SG Python API
+
+The SG Python API is not available on PyPi and must be manually installed
+from github, e.g. `pip install git+https://github.com/shotgunsoftware/python-api.git`
```

Comment:
That won't update the package if you have an older version installed. I guess the user gets an error when installing and it's up to him to understand it?